src/components/Dropdown/Dropdown.stories.jsx

```diff
@@ -10,6 +10,7 @@ import {Love} from '~/icons';
 import {dropdownData, dropdownDataGrouped, dropdownDataImages, dropdownDataTree} from '~/data';
 import IconWrapper from '~/__storybook__/IconWrapper';
 import {iconsName} from '~/__storybook__/utils';
+import {Folder, Lock} from '../../icons';
```

Comment:
To simplify the path you can use the alias `~` for `src`
```suggestion
import {Folder, Lock} from '~/icons';
```

Reply:
Done